VideoWeb/VideoWeb.Common/Models/ConsultationAnswer.cs

```diff
@@ -15,16 +15,8 @@ public enum ConsultationAnswer
         /// </summary>
         Rejected,
         /// <summary>
-        /// Cancel a consultation request
-        /// </summary>
-        Cancelled,
-        /// <summary>
         /// Consultation request failed
         /// </summary>
-        Failed,
-        /// <summary>
-        /// No consultation rooms available
-        /// </summary>
-        NoRoomsAvailable
+        Failed
```

Comment:
Other than in `PrivateConsultationParticipantsComponent` I can't see how we handle failed in the dotnet side

Reply:
updated controller to return failed if the transfer fails